lib/zipkin-tracer/zipkin_logger_tracer.rb

```diff
@@ -1,17 +1,24 @@
 require 'zipkin-tracer/zipkin_tracer_base'
 require 'zipkin-tracer/hostname_resolver'
+require 'json'
 
 module Trace
   class ZipkinLoggerTracer < ZipkinTracerBase
+    TRACING_KEY = 'Tracing information'
 
     def initialize(options)
       @logger = options[:logger]
+      @logger_accepts_data = @logger.class.name.split('::').first == 'Lorekeeper'
```

Comment:
I tried but could not make it work.
Lorekeeper uses method missing magic and `respond_to?` would respond false to any method